pom.xml

```diff
@@ -33,8 +33,9 @@
 
   <developers>
     <developer>
+      <id>google</id>
```

Comment:
I don't know what "the unique ID of the developer in the SCM" means exactly, but it could in fact be `google` given that this repo is `https://github.com/google/gson`?